core/templates/dev/head/player/ProgressDotsDirective.js

```diff
@@ -43,7 +43,6 @@ oppia.directive('progressDots', [function() {
             return;
           } else if (newValue === oldValue + 1) {
             $scope.dots.push({});
-            playerPositionService.setActiveCardIndex($scope.dots.length - 1);
```

Comment:
Oh, interesting. Yeah, it might not be necessary. The two cases to check are: the player advancing naturally, or the player clicking an existing progress dot. If both work without this line, then yep, it isn't necessary.